Tasks/GooglePlayReleaseV4/task.json

```diff
@@ -256,6 +256,14 @@
             "helpMarkDown": "Select this option to send changes for review in GooglePlay Console. If changes are already sent for review automatically, you shouldn't select this option. [More info](https://developers.google.com/android-publisher/api-ref/rest/v3/edits/commit#query-parameters).",
             "groupName": "advanced"
         },
+        {
+            "name": "rescueChangesNotSentForReview",
+            "type": "boolean",
+            "label": "Retry commit if changesNotSentForReview failed",
+            "defaultValue": false,
+            "helpMarkDown": "Retry commit if changesNotSentForReview failed",
+            "groupName": "advanced"
+        },
```

Comment:
Is this worth to add visibility rule by the changesNotSentForReview as ```"visibleRule": "changesNotSentForReview = true"```  due to they are dependent?

Reply:
Hi @ismayilov-ismayil,
you are right. Should be ok if it field is only displayed when changesNotSentForReview = true.